### ISSUE TITLE
[BUG] online documentation does not work completely with edge and 11.1 version

### ISSUE THREAD

**Issue by JiLleON** [docker-mailserver/docker-mailserver#2688]
### Miscellaneous first checks

- [X] I checked that all ports are open and not blocked by my ISP / hosting provider.
- [X] I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.

### Affected Component(s)

impossible to get acquainted with some examples from online documentation (FAQ -> How can I make SpamAssassin better recognize spam? ->  spoiler with example does not open); search does not work; quick navigation on the right does not work

### What happened and when does this occur?

```Markdown
at any time, you just need to visit online documentation (edge or 11.1) and try anything I have listed
```

### What did you expect to happen?

```Markdown
same work as with 11.0 and previous releases
```

### How do we replicate the issue?

```Markdown
1. visit https://docker-mailserver.github.io/docker-mailserver/edge/faq/#how-can-i-make-spamassassin-better-recognize-spam
2. try to open example in spoiler
----------
1. visit https://docker-mailserver.github.io/docker-mailserver/edge/
2. try to search smth with search
----------
1. visit FAQ or Introduction
2. try to quickly navigate through the headings on the right
```

### DMS version

edge and 11.1

### What operating system is DMS running on?

Other (unsupported)

### What instruction set architecture is DMS running on?

Other (unsupported)

### What container orchestration tool are you using?

Other (unsupported)

### What level of experience do you have with Docker and mail servers?

- [ ] I am inexperienced with docker
- [ ] I am inexperienced with mail servers
- [ ] I am uncomfortable with the CLI

### Code of conduct

- [X] I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
- [X] I have read the [README](https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md) and the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/) and I searched the [issue tracker](https://github.com/docker-mailserver/docker-mailserver/issues?q=is%3Aissue) but could not find a solution

### Improvements to this form?

```Markdown
separate template for issues / requests in the documentation, which does not require mandatory checkboxes about ports and ssl errors
```

**Comment by JiLleON** [comment]
I think it should be a low priority and fixed when any of you have free time for it. Also thank you for supporting this image, it made some tasks easier for me.

**Comment by polarathene** [comment]
Same as https://github.com/docker-mailserver/docker-mailserver/issues/2680#issuecomment-1183138388

I thought this was fixed with the related PR, but it seems [our workflow did not have a condition to trigger a rebuild of the docs](https://github.com/docker-mailserver/docker-mailserver/blob/master/.github/workflows/docs-production-deploy.yml) when we update the image only..

I'll resolve this tomorrow, sorry about that!

**Comment by georglauterbach** [comment]
Just checked this, the example does open for me now on `latest`. I will close this then. Ping me if you think this is not actually resolved.

**Comment by polarathene** [comment]
> Ping me if you think this is not actually resolved.

`11.1` is still broken AFAIK and would require a rebuild of docs from that tag, but with a newer docs container than we used to generate it at the time. New build would then need to replace the existing `11.1` deployment on `gh-pages` branch.

Anything prior to v12 also points to edge as the canonical link for crawlers/indexers to archive. Not as bad, and less of an issue if old docs are eventually dropped over time.

**Comment by georglauterbach** [comment]
Sounds good enough for me :D
